generators/php/codegen/src/asIs/Union.Template.php

```diff
@@ -2,20 +2,61 @@
 
 namespace <%= coreNamespace%>;
 
+use Attribute;
+
+/**
+ * Union type attribute for flexible type declarations.
+ *
+ * This class is used to define a union of multiple types for a property.
+ * It allows a property to accept one of several types, including strings, arrays, or other Union types.
+ * This class supports complex types, nested unions, and arrays, providing a flexible mechanism for property type validation and serialization.
+ *
+ * Example:
+ *
+ * ```php
+ * #[Union('string', 'int', 'null', new Union('boolean', 'float'))]
```

Comment:
Is this equivalent to the following?

```suggestion
 * #[Union('string', 'int', 'null', 'boolean', 'float')]
```

Is the format you've included here used whenever there is an undiscriminated union that references another undiscriminated union?

The main reason I ask is for the docstring included here, we might want to use the simpler, more common representation.

Reply:
as discussed offline, leaving this in for comprehensive testing purposes, but hopefully will not be seen in the wild